app/util/planParamUtil.js

```diff
@@ -197,11 +197,12 @@ export function planQueryNeeded(
       );
 
     case PLANTYPE.SCOOTERTRANSIT:
-      /* special logic: relaxed scooter query is made only if no networks allowed, and scooters are available for selection */
+      /* special logic: relaxed scooter query is made only if no networks allowed */
       return (
+        config.scooterRoutingOn &&
```

Comment:
We do not need a second  scooter switch to add unnecessary complexity. just use availableForSelection here.